README.md

```diff
@@ -1 +1,45 @@
-# charmed-kafka-rock
\ No newline at end of file
+# Charmed Kafka ROCK
+
+This repository contains the packaging metadata for creating a Charmed Kafka ROCK. This ROCK image is based on the [Charmed Kafka Snap](https://github.com/canonical/charmed-kafka-snap)  
+
+For more information on ROCKs, visit the [rockcraft Github](https://github.com/canonical/rockcraft). 
+
+## Building the ROCK
+The steps outlined below are based on the assumption that you are building the ROCK with the latest LTS of Ubuntu.  
+If you are using another version of Ubuntu or another operating system, the process may be different. 
+To avoid any issue with other operating systems you can simply build the image with [multipass](https://multipass.run/):
+```bash
+sudo snap install multipass
+multipass launch 22.04 -n rock-dev
+multipass shell rock-dev
+``` 
+
+### Clone Repository
+```bash
+git clone https://github.com/canonical/charmed-kafka-rock.git
+cd charmed-kafka-rock
+```
+### Installing Prerequisites
+```bash
+sudo snap install rockcraft --edge --classic
+sudo snap install docker
+sudo snap install lxd
```

Comment:
Doesn't this lead to issues with conflicting iptables? [Context](https://discuss.linuxcontainers.org/t/lxd-losts-iptables-rules-with-docker/15045). This might have been fixed already, so maybe it's not relevant.

Reply:
Valid point, still topical. Here is example how GH action setup-lxd handle case when lxd and docker are installed sumultanesouly: https://github.com/canonical/setup-lxd/blob/main/action.yml#L43